src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -3878,11 +3878,13 @@ public void run3() {
-        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 1), GT_Utility.getIntegratedCircuit(1)}, GT_Values.NF, CustomItemList.NeutronReflectorSmallParts.get(1), 300, 120);
-        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{CustomItemList.NeutronReflectorSmallParts.get(5L), GT_OreDictUnificator.get(OrePrefixes.plateDouble, Materials.Beryllium, 2L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorReflectorThick", 1L, 0), 600, 64);
-        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{CustomItemList.NeutronReflectorSmallParts.get(5L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.TungstenCarbide, 2L)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorReflectorThick", 1L, 0), 600, 64);
-        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_ModHandler.getModItem("IC2", "reactorReflectorThick", 1L, 0), GT_Utility.getIntegratedCircuit(1)}, GT_Values.NF, CustomItemList.NeutronReflectorParts.get(1), 600, 120);
+       //Neutron reflector recipes
+       GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.plateDouble, Materials.Copper, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Tin, 4L), GT_OreDictUnificator.get(OrePrefixes.plateAlloy, Materials.Carbon, 2L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Graphite, 32L), GT_Utility.getIntegratedCircuit(1)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 0), 1200, 480, true);
+       GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.plateDouble, Materials.Beryllium, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Tin, 4L), GT_OreDictUnificator.get(OrePrefixes.plateAlloy, Materials.Carbon, 2L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Graphite, 16L), GT_Utility.getIntegratedCircuit(1)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 0), 900, 480, true);
+       GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.plateDouble, Materials.TungstenCarbide, 1L), GT_OreDictUnificator.get(OrePrefixes.plate, Materials.Tin, 4L), GT_OreDictUnificator.get(OrePrefixes.plateAlloy, Materials.Carbon, 2L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Graphite, 8L), GT_Utility.getIntegratedCircuit(1)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 0), 400, 480, true);
+       GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.plateDouble, Materials.Beryllium, 2L), GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 1), GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 1), GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 1), GT_ModHandler.getModItem("IC2", "reactorReflector", 1L, 1), GT_Utility.getIntegratedCircuit(1)}, GT_Values.NF, GT_ModHandler.getModItem("IC2", "reactorReflectorThick", 1L, 0), 600, 1920, true);
```

Comment:
I vaguely recall there are some reflector part thing in dreamcraft, should we use them instead?

Reply:
this parts where helper items and got removed now (recipes) because we not longer use it. Before we not had a 9x input assembler